nvflare/app_opt/xgboost/histogram_based_v2/sec/server_handler.py

```diff
@@ -33,6 +40,9 @@ def __init__(self):
         self.aggr_result_to_send = None
         self.aggr_result_lock = threading.Lock()
 
+        if tenseal_imported:
+            decomposers.register()
+
```

Comment:
Should we do this in this component?
Is there a better place to auto register decomposers in app_opt?

Reply:
Yuhong is working on this. We will use his auto registering logic once he is done.